### ISSUE TITLE
Error 500 owncloud

### ISSUE THREAD

**Issue by tdecurtins** [privacyidea/privacyidea#348]
I have gone through the tutorials and once I enable pi then owncloud just gives an error 500 page.  Also in the owncloud log its giving a lot of these: Error	PHP	Call to undefined method OC_User::getUsedBackends() at /var/www/owncloud/apps/user_privacyidea/appinfo/app.php#11

**Comment by cornelinux** [comment]
ownCloud is a looong story for its own.
The authentication code in owncloud is everything else than good and modern.
And I did not find the right documentation.
I was told, that OC_ calls are internal calls which should not be used. Well, I did not read it anywhere.

So bottomline: There is no fun (and no money) in developing and maintaining the owncloud plugin.
So I do not do it.

Sorry for this negative response.
